semgrep_rules.yaml

```diff
@@ -0,0 +1,7 @@
+rules:
```

Comment:
Maybe we can make it a `.semgrep_rules.yaml` to avoid polluting repo's root dir?

Reply:
Makes sense. Thanks!